.github/workflows/release-with-changesets.yml

```diff
@@ -107,12 +103,15 @@ jobs:
         name: Install dependencies
         shell: bash
         run: npm ci
-      - if: steps.packagejson.outputs.exists == 'true'
-        name: Add plugin for conventional commits for semantic-release
-        run: npm install --save-dev conventional-changelog-conventionalcommits@5.0.0
       - if: steps.packagejson.outputs.exists == 'true'
         name: Publish to any of NPM, Github, and Docker Hub
+        uses: changesets/action@v1
         id: release
+        with:
+          publish: npm run publish-packages
```

Comment:
As per the [action documentation](https://github.com/changesets/action), don't we need to set `setupGitUser` to false? I'm confused a bit tbh